### PR TITLE
fix docs: unindent blocks to allow copy pasting more easily

### PR DESCRIPTION
**Pull request by lorenzoritter**
## Why are these changes needed?

Inconsistent indents in code samples of the Getting Started section make it harder to copy paste the code. Since this is the entry point of many users, I corrected the wrong indents so that they can copy paste the code samples more easily.

I also removed some indents in logs for consistency.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/ray-overview/getting-started.md [hunk 1]
```diff
@@ -636,56 +636,50 @@ pip install -U "ray[default]"
...
+def task_running_300_seconds():
+print("Start!")
+time.sleep(300)
 
-    # Create 2 tasks
-    tasks = [task_running_300_seconds.remote() for _ in range(2)]
+@ray.remote
+class Actor:
+def __init__(self):
+    print("Actor created")
```
Comment (aslonnie) [inline doc/source/ray-overview/getting-started.md:652]: this indentation does not look correct.

Comment (lorenzoritter) [reply doc/source/ray-overview/getting-started.md]: Good catch, thanks @aslonnie ! Updated.

**Comment by anyscalesam** [comment]
@angelinalg can you shepherd this into and merge when ready?

**Review by aslonnie — APPROVED** [review]

**Review by can-anyscale — APPROVED** [review]
thanks for the screenshot

**Pull request merged**